src/components/pages/MissionWrite/RenderUploader.js

```diff
@@ -17,7 +23,17 @@ const RenderUploader = ({ fileList, onChange, onPreview, onSubmit }) => {
       >
         {fileList.length < 5 && `Add Page ${fileList.length + 1}`}
       </Upload>
-      <button onClick={e => onSubmit(e)}>Upload</button>
```

Comment:
It looks like the Upload button got removed in your edits

Reply:
oops

Reply:
it's fixed for my next merge